src/libcalamaresui/modulesystem/PythonQtViewModule.cpp

```diff
@@ -31,7 +31,7 @@
 #include "JobQueue.h"
 
 #include <PythonQt.h>
-#include <extensions/PythonQt_QtAll/PythonQt_QtAll.h>
+#include <PythonQt_QtAll.h>
```

Comment:
I think this will need more CMake support: if PythonQt_QtAll.h has moved (is that a packaging decision, or something changed upstream in a later version than what I see?), we should look for it in CMake and add adequate include paths.